tests/e2e-leg-10/sandbox-basic/60-assert.yaml

```diff
@@ -11,6 +11,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 900
```

Comment:
I remember the default timeout for all job is 1000, any reason we want to reduce the time out here?

Reply:
The default is 600s in kuttl-test.yaml